app/code/Magento/Wishlist/Block/Customer/Wishlist.php

```diff
@@ -78,14 +83,63 @@ protected function _prepareCollection($collection)
     }
 
     /**
-     * Preparing global layout
+     * Paginate Wishlist Product Items collection
      *
      * @return void
      */
+    public function paginateCollection()
+    {
+        $page = $this->getRequest()->getParam("p", 1);
+        $limit = $this->getRequest()->getParam("limit", 10);
+        $this->_collection
+            ->setPageSize($limit)
+            ->setCurPage($page);
+    }
+
+    /**
+     * Retrieve Wishlist Product Items collection
+     *
+     * @return \Magento\Wishlist\Model\ResourceModel\Item\Collection
+     */
+    public function getWishlistItems()
```

Comment:
Please change to private function

Reply:
please check 
Access level to Magento\Wishlist\Block\Customer\Wishlist::getWishlistItems() must be public (as in class Magento\Wishlist\Block\AbstractBlock) in /home/users/ratnesh/www/html/magento/m2.3-1/app/code/Magento/Wishlist/Block/Customer/Wishlist.php on line 276

not suggested to do so